### ISSUE TITLE
Initialization of the ClarityModule in jasmine tests should not consume more than a fraction of the jasmine default timeout duration.

### ISSUE THREAD

**Issue by krotscheck** [vmware/clarity#483]
```
[ x ] enhancement
```

### Expected behavior
Initialization of the ClarityModule in jasmine tests should not consume more than a fraction of the jasmine default timeout duration.

### Actual behavior
We've observed initialization times of Clarity ranging from 1.2 seconds to 4 seconds in Firefox, using the below attached unit test. The jasmine default timeout is 5 seconds - in a perfect world, we'd expect that initialization should take less than a second in all browsers.

### Reproduction of behavior
```
// Output from this test, run in Firefox
//   Bootstrap Test
//    with ClarityModule
//      ✓ should create a component (2452ms)
//    with FormsModule
//      ✓ should create a component (130ms)
//    with no modules
//      ✓ should create a component (45ms)


import {TestBed} from '@angular/core/testing';
import {Component} from '@angular/core';
import {ClarityModule} from 'clarity-angular';
import {FormsModule} from '@angular/forms';

@Component({
  template: '<h1>Hello world</h1>'
})
class TestComponent {
}

fdescribe('Bootstrap Test', function () {

  describe('with ClarityModule', function () {
    beforeEach(() => {
      TestBed.configureTestingModule({
        imports: [ClarityModule],
        declarations: [TestComponent]
      }).compileComponents();
    });

    it('should create a component', () => {
      TestBed.createComponent(TestComponent);
      expect(true).toBeTruthy();
    });
  });

  describe('with FormsModule', function () {
    beforeEach(() => {
      TestBed.configureTestingModule({
        imports: [FormsModule],
        declarations: [TestComponent]
      }).compileComponents();
    });

    it('should create a component', () => {
      TestBed.createComponent(TestComponent);
      expect(true).toBeTruthy();
    });
  });

  describe('with no modules', function () {
    beforeEach(() => {
      TestBed.configureTestingModule({
        imports: [],
        declarations: [TestComponent]
      }).compileComponents();
    });

    it('should create a component', () => {
      TestBed.createComponent(TestComponent);
      expect(true).toBeTruthy();
    });
  });
});
```

### Environment details

* **Angular version:** 2.4.5

* **Clarity version:** 0.8.5

* **OS and version:** OSX Sierra

* **Browser:** [all]

**Comment by youdz** [comment]
There was some discussion about this here: https://github.com/angular/angular/issues/12409
People had the same issue importing `MaterialModule.forRoot()`. It's actually the same issue for any components library. If the Angular team does not provide a way to import a module but only compile the actually used components, we'll need to export submodules that people can import for their unit tests: `AlertsModule`, `DatagridModule`, ...

**Comment by youdz** [comment]
Closed by #880. You can now just import the module you need for the test.
